peekaboo/toolbox/cortex.py

```diff
@@ -56,12 +57,31 @@ def __init__(self, analyzer):
 
 class CortexAnalyzerReport:
     """ Cortex analyzer report base class. """
+
+    report_schema_taxonomies = schema.Schema([{
+        "level": schema.Schema(schema.Or("info", "malicious", "safe")),
```

Comment:
See https://github.com/scVENUS/PeekabooAV/pull/189/commits/3b7ba073e747b09947475994111f1055b387082b for how I used it for the Cuckoo report. Maybe you can get some more ideas there. For example I found that both instances of nesting of `schema.Schema()` here should be redundant if in the end you're passing it to `schema.Schema()` once more to build `self.report_schema`.

Reply:
Addressed in recent commit.